js/interface.js

```diff
@@ -1,78 +1,176 @@
-var $folderContents = $('#folder-contents');
+// VAR SETUP //
+var $folderContents = $('.file-table-body');
+var $folderList;
+var $organizationAppList = $('.dropdown-menu-holder ul');
 var templates = {
   file: template('file'),
-  folder: template('folder')
+  folder: template('folder'),
+  folderItem: template('folder-item'),
+  organizations: template('organizations'),
+  apps: template('apps')
 };
+
 var currentFolderId;
 var currentFolders;
 var currentFiles;
+var counter;
```

Comment:
The counter is to get the first organization element that is appended. Didn't know a better way to do that, because I want to add a class to the first element after being appened.